process.go

```diff
@@ -0,0 +1,33 @@
+package ebpf
+
+import (
+	"fmt"
+
+	"golang.org/x/sys/unix"
```

Comment:
Please import `github.com/cilium/ebpf/internal/unix` instead, that contains dummy definitions so that working on the code base works on non-linux OS.

Reply:
Good call, I had no idea there was one. Done.